obspy/clients/eida/__init__.py

```diff
@@ -44,7 +44,7 @@
 Add debug=True to see what is going on.
 
 >>> from obspy.clients.eida import Client
->>> credentials = {"http://service.iris.edu/fdsnws/dataselect/1/queryauth":
+>>> credentials = {"http://service.iris.edu/fdsnws/dataselect/1/queryauth": \
                    ("nobody@iris.edu", "anonymous")}
```

Comment:
`...` instead of `>>>` on the continuation line would be a bit nicer, then the ` \` is not needed. But only cosmetics, of course..